machine/quanta/quanta_common_rglbmc/busybox/patches/onie-sys-eeprom-platform.patch

```diff
@@ -0,0 +1,609 @@
+Add support for onie-syseeprom and mb_detect
+
+diff --git a/include/applets.src.h b/include/applets.src.h
+index 71b8cbd..891fb67 100644
+--- a/include/applets.src.h
++++ b/include/applets.src.h
+@@ -345,6 +345,7 @@ IF_WHOAMI(APPLET_NOFORK(whoami, whoami, BB_DIR_USR_BIN, BB_SUID_DROP, whoami))
+ IF_YES(APPLET_NOFORK(yes, yes, BB_DIR_USR_BIN, BB_SUID_DROP, yes))
+ IF_ZCIP(APPLET(zcip, BB_DIR_SBIN, BB_SUID_DROP))
+ IF_SYS_EEPROM(APPLET_ODDNAME(onie-syseeprom, onie_syseeprom, BB_DIR_USR_BIN, BB_SUID_DROP, onie_syseeprom))
++APPLET_ODDNAME(mb_detect, mbdetect, BB_DIR_USR_BIN, BB_SUID_DROP, mbdetect)
```

Comment:
do you really need to alias this command on a completely new board?
there should not be a backward compatibility problem here.